FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.12.0
+- [fixed] Fix issue of real-time listeners not being properly removed. (#11458)
+- [fixed] Fix real-time fetches not being able to fetch the latest template. (#11465)
```

Comment:
Flyby comment: it might help to be more specific here since IIUC the issue a race condition that doesn't impact most users. Maybe "Fix an issue where an in-progress fetch request prevents a real-time update."

Reply:
I'm not sure that's accurate, as the reporter. This was typically triggered immediately when it happened, and on app startup I think it would be common to do a fetchAndActivate + subscribe a listener as a coding pattern

Perhaps that is unnecessary in a purely realtime-reliant system as the listener will issue a callback if there are unactivated changes but there may be a lot of existing code where the add remote-config realtime vs change completely to it

As an SDK just one layer up we're doing both in e2e testing so it bites us regardless, I just think it might be common in the wild too.

Reply:
Hmm - on re-review of your comment and my comment - your suggestion of more specific handles both cases though, so I think my comment stands as context but - your suggestion is an improvement

Reply:
updated comment to be more specific